### PR TITLE
Add indexer status

### PR DESCRIPTION
**Pull request by melMass**
* feat: ✨ First implementation of the indexer status

Only shows a red warning when the indexer is out of sync.

image
image

### CODE REVIEW

File: src/data/hicdex.js [hunk 1]
```diff
@@ -205,12 +205,6 @@ export async function fetchGraphQL(operationsDoc, operationName, variables) {
     }
   );
 
-  console.log(JSON.stringify({
```
Comment (melMass) [inline src/data/hicdex.js:198]: @xat @kylegrover Not sure if you see objections to this but it's polluting the console

Comment (xat) [reply src/data/hicdex.js:198]: no objections :)

**Comment by melMass** [comment]
Changed the alignment in 422cfd052c62767db9a69c9be2898042562f6c0e

## On desktop
image

## On mobile
image

**Comment by xat** [comment]
looks good to merge, from my point in view.

nice job! 👍

**Comment by xat** [comment]
@melMass not sure if it's intended, but the pr is still in draft mode

**Review by xat — APPROVED** [review]
great work :)

**Pull request merged**